data/src/main/java/de/ph1b/audiobook/data/repo/internals/migrations/Migration43to44.kt

```diff
@@ -28,11 +28,12 @@ class Migration43to44 : IncrementalMigration(43) {
   }
 
   private fun createNewTables(db: SupportSQLiteDatabase) {
-    db.execSQL("CREATE TABLE `bookmark` (`file` TEXT NOT NULL, `title` TEXT NOT NULL, `time` INTEGER NOT NULL, `id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL)")
-    db.execSQL("CREATE TABLE `chapters` (`file` TEXT NOT NULL, `name` TEXT NOT NULL, `duration` INTEGER NOT NULL, `fileLastModified` INTEGER NOT NULL, `marks` TEXT NOT NULL, `bookId` TEXT NOT NULL, `id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL)")
+    db.execSQL("CREATE TABLE IF NOT EXISTS `bookmark` (`file` TEXT NOT NULL, `title` TEXT NOT NULL, `time` INTEGER NOT NULL, `id` INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL)")
```

Comment:
Let's remove all the `IF NOT EXISTS`. If it already existed, we made a mistake in our code.